contracts/VToken.sol

```diff
@@ -868,31 +868,32 @@ contract VToken is
 
         uint256 redeemTokens;
         uint256 redeemAmount;
+
         /* If redeemTokensIn > 0: */
         if (redeemTokensIn > 0) {
             /*
              * We calculate the exchange rate and the amount of underlying to be redeemed:
              *  redeemTokens = redeemTokensIn
-             *  redeemAmount = redeemTokensIn x exchangeRateCurrent
              */
             redeemTokens = redeemTokensIn;
             redeemAmount = mul_ScalarTruncate(exchangeRate, redeemTokensIn);
```

Comment:
We should remove this line from here, as we move it outside the if else block at [line#892](https://github.com/VenusProtocol/isolated-pools/pull/247/files#diff-74967185a2c5eedfa0a1e0b396c10b82321afb533d1fca50ca5d896e81915e17R892) @narayanprusty